### PR TITLE
chore: add eslint and priettier config

### PR DESCRIPTION
**Pull request by kml1990**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  "rules": {
+    "linebreak-style": "off",
+    "prettier/prettier": [
+      "error",
+      {
+        "endOfLine": "auto"
+      }
+    ]
+  }
+}
```
Comment (sandram92) [inline .eslintrc.json]: nit: Can you add line break

File: src/App.tsx [hunk 2]
```diff
@@ -1,17 +1,20 @@
-import { useState } from 'react'
-import logo from './logo.svg'
-import './App.css'
+import { useState } from "react";
```
Comment (sandram92) [inline src/App.tsx]: can we change the rule for single quotes for preatier

**Review by sandram92 — CHANGES_REQUESTED** [review]
Well done Kamil. I have left some comments for you to look at

File: src/App.tsx (hunk 2 above)
Comment (kml1990) [reply src/App.tsx]: Yes, good idea 👍

File: .eslintignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+*.css
+*.svg
```
Comment (sandram92) [inline .eslintignore]: nit: add a blank line to the end of the file, please.

File: .prettierignore [hunk 4]
```diff
@@ -0,0 +1,2 @@
+**/node_modules
+**/dist
```
Comment (sandram92) [inline .prettierignore]: nit: add a blank line to the end of the file, please

**Review by sandram92 — COMMENTED** [review]
Great job Kamil

**Review by sandram92 — APPROVED** [review]

**Pull request merged**